src/commands/account/delete_key/use_publickeylist_type/mod.rs

```diff
@@ -0,0 +1,32 @@
+use interactive_clap::ToCli;
+
+#[derive(Debug, Clone)]
+pub struct PublicKeyList(pub Vec<crate::types::public_key::PublicKey>);
```

Comment:
I think you can use `near_crypto::PublicKey` here directly. There is no point in using the wrapper type. And then you don't need to repackage the items at all: https://github.com/near/near-cli-rs/pull/207/files#diff-a637fdd677d48fb8d9a4460b98e3ed6a82e647bebbf16435e42e1e667f829ef0R35-R37